apps/main/src/llamalend/entities/llama-markets.ts

```diff
@@ -38,6 +38,7 @@ export type LlamaMarket = {
   address: Address
   controllerAddress: Address
   assets: Assets
+  maxLtv: number
```

Comment:
feels like we're doing something wrong when adding a single attribute takes 56 lines 😵

Reply:
Modifying 4 endpoints, adding the column itself, setting visibility, sorting behavior, a new cell component. I don't think it can be shortened 🤔